projects/pastanaga-angular/src/lib/date-picker/date-picker.component.scss

```diff
@@ -74,20 +74,20 @@
         display: flex;
         align-items: center;
         color: var(--header-color);
-        font-size: 12px;
+        font-size: font-size(xs);
         font-weight: 700;
-        line-height: 16px;
+        line-height: line-height(s);
         text-transform: uppercase;
```

Comment:
seems like you could have used pastanaga typography class `.overline` here.
You can see all the typography classes in https://github.com/plone/pastanaga-angular/blob/a09a31d30635fc381ae7aeb0537052fef7aa92e6/projects/pastanaga-angular/src/styles/_typography.scss